doc/manpages/mapping.asciidoc

```diff
@@ -0,0 +1,52 @@
+KAKOUNE(1)
+==========
+
+NAME
+----
+mapping - a
+
+Description
+-----------
+Creating and removing shortcuts boils down to the following functions,
+respectively:
+
+---------------------------------------
+map [flags] <scope> <mode> <key> <keys>
+unmap <scope> <mode> <key> [<expected>]
+---------------------------------------
+
+The *map* command makes *key* behave as if the *keys* sequence was typed.
+
+*mode* dictates in what context the mapping will be available:
+
+	*insert*::
+		insert mode (default: 'i')
+	*normal*::
+		normal, commands mode
+	*prompt*::
+		prompts, inputs (e.g. '/')
+	*menu*::
+		multiple items dropdown menus (e.g. ':')
```

Comment:
I really had no idea what the difference was with `prompt`, so I used my best judgement. Also I put default keys on the other mode explanations because I put one for this mode, resulting in all the above remarks.